anycast/docs/ProtoOnpremHostRef.md

```diff
@@ -0,0 +1,186 @@
+# ProtoOnpremHostRef
+
+## Properties
+
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**Id** | Pointer to **int64** |  | [optional] [readonly] 
```

Comment:
Please update the generated client to remove this `readonly`

Reply:
done